source/ui/AMToolButton.h

```diff
@@ -7,22 +7,32 @@ class AMToolButton : public QToolButton
 {
 	Q_OBJECT
 
+	Q_PROPERTY(ColorState colorState READ colorState WRITE setColorState NOTIFY colorStateChanged)
```

Comment:
What is the use of the QProperty here? I know they're more general than the way we usually use them with AMDbObject, but what is it doing here?